### PR TITLE
Form

### PR DESCRIPTION
**Pull request by aliveGUY**
# Finishing mobile version with a team
## Main changes:
* **Add markup for From section**
* **Add styles for Form section**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,206 +1,256 @@
...
+              <input class="Form-input" type="email" placeholder="Email address" name="user_email" />
+            </li>
+            <li>
+              <textarea name="user_message">Enter text here</textarea>
+            </li>
+            <li class="Form-button">
+              <button type="submit" class="button-reg">Get in touch</button>
+            </li>
+          </ul>
+        </form>
```
Comment (smunozmo) [inline index.html:208]: - [ ]  Your form is looking great! However, the `name` field is missing validation, it should accept up to 30 characters.

- [ ]  You are almost there! The `message` field also requires validation, it should accept up to 500 characters. 

- [ ]  All three fields are almost ready, just make them a `required` so the form cannot be sent empty.

File: index.css [hunk 2]
```diff
@@ -378,3 +377,87 @@ body {
   font-size: 22px;
   line-height: 43px;
 }
+
+.Form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 25px;
+}
```
Comment (smunozmo) [inline index.css:386]: Good job using Flexbox to layout your form, it looks awesome!

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @aliveGUY, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aliveGUY** [comment]
## Main changes: 
:heavy_check_mark: **Add validation for name**
:heavy_check_mark: **Add validation for message**
:heavy_check_mark: **Add required for form**

File: index.html [hunk 3]
```diff
@@ -1,206 +1,256 @@
...
+        <form action="https://formspree.io/f/mqknlvyq" method="post" class="Form-form">
+          <ul class="Form-list">
+            <li>
+              <input class="Form-input" type="text" placeholder="Full name" name="user_name" required maxlength="30"/>
+            </li>
+            <li>
+              <input class="Form-input" type="email" placeholder="Email address" name="user_email" required/>
+            </li>
+            <li>
+              <textarea name="user_message" required maxlength="500">Enter text here</textarea>
```
Comment (BenMukebo) [inline index.html:202]: - [Optional ] This field should contain a place holder I strongly recommend you make this change, please remove the text value and replace it with  place holder, it should be something like 
```
<textarea placeholder="Enter your message here"  name="user_message" required maxlength="500"></textarea>
```

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @aliveGUY ** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**